AWS/legos/aws_find_redshift_clusters_with_low_cpu_utilization/README.md

```diff
@@ -0,0 +1,23 @@
+[<img align="left" src="https://unskript.com/assets/favicon.png" width="100" height="100" style="padding-right: 5px">]
+(https://unskript.com/assets/favicon.png)
+<h1>AWS Find Redshift Clusters with low CPU Utilization</h1>
+
+## Description
+Find underutilized Redshift clusters in terms of CPU utilization.
```

Comment:
This is not following the naming convention we outlined [here](https://unskript.atlassian.net/wiki/spaces/ENGINEERIN/pages/702545936/Runbook+Guidelines#Naming-convention). Use get instead of find